client/src/main/java/playing/FieldPosition.java

```diff
@@ -0,0 +1,21 @@
+package playing;
+
+public class FieldPosition {
+
+
+    private final Integer x;
+    private final Integer y;
+
+    FieldPosition(double x, double y) {
+        this.x = (int) x;
```

Comment:
why? if you store Integer why constructor has doubles?

Reply:
Becouse x and y that is passed to constructor is from Gui Rectangle class that stores this values ass doubles